src/SprykerMiddleware/Shared/Process/ProcessConfig.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerMiddleware\Shared\Process;
+
+interface ProcessConfig
```

Comment:
With this interface you hard couple users to it.
Please use a class and extend abstract shared config.